docs/src/pages/landing/Pro.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import classNames from 'classnames';
```

Comment:
Do we still have this in dependencies of the docs? Should we drop it if this was the last usage?

Reply:
I think one package had it as a peer dependency. So if you remove it, make sure no additional "missing peer dependency" warnings are logged.

Reply:
I can't find `classnames` listed as a dependency in our pakages.json. It seems we were relying on a transitive dependency.